credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt

```diff
@@ -94,40 +101,75 @@ class VerifiableCredentialTest {
   }
 
   @Test
-  fun `signing works`() {
-    val keyManager = InMemoryKeyManager()
-    val issuerDid = DidKey.create(keyManager)
-    val holderDid = DidKey.create(keyManager)
-
-    val vc = VerifiableCredential.create(
-      type = "StreetCred",
-      issuer = issuerDid.uri,
-      subject = holderDid.uri,
-      data = StreetCredibility(localRespect = "high", legit = true)
+  fun createJwk() {
+    val privateJwk = JWK.parse(
+      """{
+          "kty": "EC",
+          "d": "OeBCt3M8roz9F9Ny192TtZXf_-qPzAkwSM6ep7QdSO4",
+          "use": "sig",
+          "crv": "secp256k1",
+          "kid": "NB_qlUIr06-AikVVMFreq0lc-omQtzc6lwhhcvgO6r4",
+          "x": "DdtN8W6x_34pB_nkxR0e1tmDkNnsJeusBAEPzKWgf_Y",
+          "y": "u3W135inodLqtcEb9jNGS3JsM_uFKmkJSb8Trc9luWI",
+          "alg": "ES256K"
+        }"""
     )
+    val publicKey = privateJwk.toPublicJWK()
+    val base64Encoded = Convert(publicKey.toJSONString()).toBase64Url(padding = false)
 
-    val vcJwt = vc.sign(issuerDid)
-    val parsedJwt = SignedJWT.parse(vcJwt) // validates JWT
+    val did = "did:jwk:$base64Encoded"
+    println(did)
+  }
 
-    val didDocument = issuerDid.resolve().didDocument
-    val assertionMethod = didDocument.assertionMethodVerificationMethodsDereferenced.first()
+  data class TestVectors(
+    val description: String,
+    val vectors: List<TestVector>
+  )
 
-    val jwtHeader = parsedJwt.header
-    assertNotNull(jwtHeader.algorithm)
-    assertEquals(JOSEObjectType.JWT, jwtHeader.type)
-    assertEquals(assertionMethod.id.toString(), jwtHeader.keyID)
+  data class TestVector(
+    val description: String,
+    val input: TestInput,
+    val output: String?,
+    val errors: Boolean?,
+  )
 
-    val jwtClaims = parsedJwt.jwtClaimsSet
-    assertNotNull(jwtClaims.issueTime)
-    assertNotNull(jwtClaims.getClaim("vc"))
-    assertEquals(issuerDid.uri, jwtClaims.issuer)
-    assertEquals(holderDid.uri, jwtClaims.subject)
+  data class TestInput(
+    val signerDidUri: String?,
+    val signerPrivateJwk: Map<String, Any>?,
+    val credential: Map<String, Any>?,
+  )
 
-    val vcDataModelJson = jwtClaims.getJSONObjectClaim("vc")
-    val vcDataModel = VcDataModel.fromMap(vcDataModelJson)
+  @Test
+  fun `creates test vectors`() {
```

Comment:
Slightly misleading name, in that, it could be interpreted that this function creates the test vectors JSON

Maybe...

```suggestion
  fun `test vectors: creates`() {
```

Reply:
Unforturnately, cannot include `:` chars in the test names. Instead, I went with the approach of defining a new class as shown below. Open to suggestions/feedback!

```kt
class TestVectorsCredentialsTest {
  @Test
  fun creates() {
// ...
  }
}
```